update interactive tutorial for language and links (#193)
qdrant/qdrant-web-ui: src/components/InteractiveTutorial/MdxPages/Index.mdx

# Interactive Tutorial

The quickest way to see how Qdrant works is to access its API from this live instance.

You will use the [Qdrant REST API](https://api.qdrant.tech) to interact with sample data. Here you can perform various operations such as inserting vectors, searching, and applying filters.

**Note:** All created collections and vectors will remain in the cluster until you delete them. You can always return to the tutorial and edit code blocks to get different results.

|Tutorial|What you will learn|
|-|-|
|[Quickstart](#/tutorial/quickstart)|Create a collection, upsert vectors & run a search.|
|[Payload Filtering](#/tutorial/filtering-clauses)|Refine search results based on payload conditions.|

## Next steps:

Once you are ready to leave this sandbox tutorial, you can try the complete [REST API](https://api.qdrant.tech) from the **Console**. All your resources will be persisted. 

> To begin building a prototype application, you should [setup Qdrant on Docker](https://qdrant.tech/documentation/quick-start/) and start using one of our [language-specific Clients](https://qdrant.tech/documentation/interfaces/).

Reviewer: ```suggestion
You can use one of our [language-specific Clients](https://qdrant.tech/documentation/interfaces/) to build your applications.
```

They're already running Qdrant. So redundant.
